recipe/bld.bat

```diff
@@ -0,0 +1,9 @@
+:: We set the version in the pin file but the build expects this to be a directory.
+:: we should wither patch it or send a PR upstream to avoid this workaround.
+set hdf4=
+
+set LIBRARY_DIRS=%LIBRARY_BIN%;%LIBRARY_LIB%
+set INCLUDE_DIRS=%LIBRARY_INC%
+
+REM %PYTHON% -m pip install . --no-deps -vv
+%PYTHON% -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```

Comment:
The [numpy build script](https://github.com/conda-forge/numpy-feedstock/blob/master/recipe/bld.bat) has this:
```
%PYTHON% -m pip install --no-deps --ignore-installed .
```
maybe the dot should be at the end?

Reply:
I just tested. That's not the issue but it wouldn't hurt to correct it anyway.

Note that you can [set up an rdp connection](https://www.appveyor.com/docs/how-to/rdp-to-build-worker/) to appveyor `on_finish` for debugging. I was able to build pyhdf just fine without conda build, by just activating the miniconda environment, installing numpy and hdf4, and running the pip install command. So, it appears that issue is with `conda build`.

Reply:
BTW, the `dot` was moved from the end to the beginning to avoid confusion in our docs. So people don't think it was just an end of the sentence.

Indeed looks like something specific to our build here.